client/cluster.go

```diff
@@ -219,6 +219,10 @@ func (c *ClusterClient) Join(name string) (api.MachineState, error) {
 		return machineState, fmt.Errorf("error joining cluster: %w", err)
 	}
 
+	if err := c.ansible.Infrustructure(); err != nil {
```

Comment:
The other place where this is run it looks like this:
```
	if c.cluster.CloudProvider() == api.Safespring ||
		c.cluster.CloudProvider() == api.CityCloud {
		if err := c.ansible.Infrustructure(); err != nil {
			return fmt.Errorf("error infrastructure: %w", err)
		}
	}
```

Perhaps it's best to to the same for these two?

Reply:
Fixed now